official/wide_deep/wide_deep.py

```diff
@@ -193,7 +193,6 @@ def parse_csv(value):
   features, labels = iterator.get_next()
   return features, labels
 
-
```

Comment:
Let's restore this newline. We still want two blank lines between top-level functions.

Reply:
Thanks for catching this!